src/pages/api/github/pr.ts

```diff
@@ -120,12 +121,12 @@ async function createForkAndPR(
   const prTitle = `Add ${fileSlug} to ${directoryPath}`;
   const prDescription = `This PR adds [${fileName}](${metaData.source}) transcript to the ${directoryPath} directory.`;
   const prResult = await octokit.request("POST /repos/{owner}/{repo}/pulls", {
-    owner: forkOwner, // update to upstreamOwner once tested
-    repo: forkRepo,
+    owner: prRepo === "user" ? forkOwner : upstreamOwner, // update to upstreamOwner once tested
+    repo: prRepo === "user" ? forkRepo : upstreamRepo,
     title: prTitle,
     body: prDescription,
     head: `${forkOwner}:${newBranchName}`,
-    base: baseBranchName,
+    base: prRepo === "user" ? baseBranchName : "master", // get upstream base for btc transcript dynamically?,
```

Comment:
baseBranchName is coming from the fork. The forked repo carries over the default branhc from the source. We can test this to be sure. But basebranchName works fine.

Reply:
Should we test by raising against btctanscript repo?

Reply:
Mmm, I think Jonas would have the say on that.